Spanner/src/Connection/Grpc.php

```diff
@@ -455,6 +457,43 @@ public function createSession(array $args)
         ]);
     }
 
+    /**
+     * Note: This should be removed once GAPIC exposes the ability to execute
+     * concurrent requests.
+     *
+     * @access private
+     * @experimental
+     * @param array $args
+     * @return PromiseInterface
+     * @throws \Google\ApiCore\ValidationException
```

Comment:
What is the condition which causes this exception to bubble up?

Reply:
It looks like PHPStorm identified it bubbling up as a possible exception from GRPC when I generated the docblock. I don't think there's a valid use case for end users where this would be exposed; cleaning it up.